.github/workflows/publish-docker-hub.yaml

```diff
@@ -15,5 +15,4 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
           repository: unitytechnologies/datasetinsights
-          tags: latest
```

Comment:
After this change, will we see the latest tag when we merge to the current master?

Reply:
This is merge request in 0.2.x branch. Master will have different code with latest tag.

Reply:
I am not sure if this will work. How do you test this out?

Reply:
When this GitHub action runs, it publish two docker image, one with latest tag (tag_with_ref: true), other with tag name(tags: latest).
- Keeping only tag_with_ref: true, will publish only one docker image with released tag. 

Reference: https://docs.github.com/en/actions/guides/publishing-docker-images

Reply:
How does this change in a branch fix the issue? Have you tested to make sure it will work as desired?

Reply:
- If we release from this branch it should publish only one docker with the release tag and this is what we want to achieve. With tag_with_ref: true,  only one docker image get published with release tag as per above documentation. 
- How to test this before merging the code:
1. One option is I can release from this PR branch with some test tag but it seems risky.
 2. Create a test project on docker hub and run this git hub action.
Please let me know which is preferred or another way would also work?

Reply:
Let's test it out using approach 2 to see if this change actually meet the desired behavior.

Reply:
Repo: https://github.com/sanjayuconn/ML_Deployment/tree/sanjayuconn-patch-1

- During the release(action trigger) the branch we select that's github yaml get triggered.

Reply:
https://github.com/sanjayuconn/literate-broccoli/tags

Reply:
excellent! This confirmed your hypothesis that a tag release on feature branch will trigger github action execution in a feature branch.